src/BUILD

```diff
@@ -630,7 +630,12 @@ cc_binary(
     ],
     deps = [
         "//src:ovms_lib",
-    ],
+    ] + select({
+            "//conditions:default": [
+                "//src:ovmscalculatoroptions_cc_proto", # ovmscalculatoroptions_proto - just mediapipe stuff with mediapipe_proto_library adding nonvisible target
```

Comment:
Check:
https://github.com/openvinotoolkit/model_server/pull/1953/files#diff-b2d5f1b40d3ad56ddc7cb7ce51a06491c5b8a22eeb85a3cf2093b651ee7fce28R879
we need to include calculators not only generated proto files.
Additionally we should remove  dependency from L881 from ovms_test since it should use calculators directly from OVMS.

Reply:
IF we want to use it. But I understand we want to use @mediapipe ovms calculators here?